js/widget.js

```diff
@@ -1,4 +1,4 @@
-import A from "https://esm.sh/aladin-lite@3.3.3-beta";
+import A from "https://esm.sh/aladin-lite@3.4.0-beta";
```

Comment:
You might be able to remove the goToRaDec after initialization with this bump.

Reply:
Also, this should be in the changelog and in the readme

Reply:
Despite the version change, the removing of gotoRaDec call at the initialization re-introduce the patched bug.

Reply:
Too bad, thanks for testing

Reply:
In the correspondence table at the end of the README, could you add a line with `unreleased || 3.4.0-beta`?

Reply:
It is because when you instanciate aladin with:
```js
let aladin = Aladin({target: '0 0', cooFrame: 'galactic', ...})
```
as there is no frame given with the target position string, it will take that position as what is given in the cooFrame.

@tboch - Should we consider that behaviour as okaish or `target` should be always considered as given in ICRS frame ?

Reply:
> In the correspondence table at the end of the README, could you add a line with `unreleased || 3.4.0-beta`?

Done !